src/python/CRABClient/JobType/PrivateMC.py

```diff
@@ -31,7 +31,7 @@ def run(self, *args, **kwargs):
             self.logger.debug("LHESource found in the CMSSW configuration.")
             configArguments['generator'] = getattr(self.config.JobType, 'generator', 'lhe')
 
-            major, minor, fix = os.environ['CMSSW_VERSION'].split('_')[1:4]
+            major, minor, _ = os.environ['CMSSW_VERSION'].split('_')[1:4]
```

Comment:
We can use something like `_dummy`, `_dummy_fix` or `_unused`, `_unused_fix` etc, but I think the code here simple enough without the need of any extra explanations.